### PR TITLE
Disable Cypress's video feature

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)
- [x] Bug Fix

## Description
Disable Cypress's video feature to saves ~8 minutes!!

## Related Tickets & Documents
n/a

## QA Instructions, Screenshots, Recordings
n/a
## Added tests?
- [x] No, and this is why: CI related work

## [Forem core team only] How will this change be communicated?
- [x] This change does not need to be communicated, and this is why not: This is a CI change.

## [optional] Are there any post deployment tasks we need to perform?
n/a

### CODE REVIEW

File: cypress/integration/goToHomePage.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+describe('User Login', () => {
```
Comment (nickytonline) [inline cypress/integration/goToHomePage.js:1]: One tiny test.

File: cypress/integration/goToHomePage.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  it('Visit the homepage', () => {
+    // Go to home page
+    cy.visit('/');
```
Comment (nickytonline) [inline cypress/integration/goToHomePage.js:4]: One tiny test.

**Comment by maestromac** [comment]
`bin/e2e-ci` took only a minute to complete :thinking: . I think Cypress's result on time to complete may not factor in setup/tear down.

**Review by vaidehijoshi — APPROVED** [review]
TIL: our cypress setup ~has~ had a video feature? 🙊

**Review by atsmith813 — APPROVED** [review]
👏🏽

**Comment by maestromac** [comment]
@vaidehijoshi yeah it records all test and output to mp4. I believe there was one occasion we retrieved it to debug things.

**Pull request merged**